content/events/2022-04-26-impact-data/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Generating software containers"
+date: '2022-04-26'
+days: 5
+tease: "Workshop at IMPaCT-Data"
+continent: EU
+location: "INB, Online, Barcelona, Spain"
+location_url: "https://inb-elixir.es/"
+gtn: false
+contact: "Björn Grüning"
+tags: [ ]
```

Comment:
container?

Reply:
I think that the tags are defined in the config.json and useful to create dynamic pages, e.g. papercuts, roundtables, community calls...